CommandLine/testing/SimpleTests/math_test.sog/create.edl

```diff
@@ -0,0 +1,23 @@
+float f = (float) -.5;
+double d = -.5;
+// long double ld = -.5;
+int i = -30;
+unsigned u = 30;
+long l = -60;
+unsigned lu = 60;
+variant v = -120;
+var a = -150;
+
+cons_show_message("Test start!");
+gtest_assert_eq(abs(f),  -f);
```

Comment:
I think real numbers should be in the results column here so we don't end up in a 0 == 0 situation without realizing it